packages/@sanity/portable-text-editor/tsdoc.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Necessary to get the comments in vercelStegaCleanAll.ts to not scream about the `@hidden` and `@todo` tags